server/fishtest/stats/stat_util.py

```diff
@@ -310,6 +311,44 @@ def update_SPRT(R, sprt):
     elif sprt["llr"] > sprt["upper_bound"] - o1:
         sprt["state"] = "accepted"
 
+def residuals(coeffs, table):
+    # If k1j,..,ktj is a row in a contingency table
```

Comment:
Textbooks usually use this convention for contingency table:
 - table indices [row, column]: `Kij`
 - row sum `Ki.`
 - column sum `K.j`
 - total sum `K..`

See [this handout](http://fisher.stats.uwo.ca/faculty/kulperger/SS3858/Handouts/ContingencyTables.pdf) that follows "Categorical Data Analysis" by Agresti.

Reply:
Makes sense! I will fix it.